docs/upgrade.html

```diff
@@ -137,6 +137,8 @@ <h5><a id="upgrade_400_notable" href="#upgrade_400_notable">Notable changes in 4
                     <ul>
                         <li>The <code>enable.idempotence</code> configuration will no longer automatically fall back when the <code>max.in.flight.requests.per.connection</code> value exceeds 5.
                         </li>
+                        <li>The deprecated <code>sendOffsetsToTransaction</code> method has been removed from the Producer API.
```

Comment:
Could you please use `sendOffsetsToTransaction(Map<TopicPartition, OffsetAndMetadata>, String)` instead?

Reply:
> Could you please use `sendOffsetsToTransaction(Map<TopicPartition, OffsetAndMetadata>, String)` instead?

Sure.